### DISCUSSION TITLE
Welcome to Nitin-Fashionwear Discussions!

### DISCUSSION THREAD

**Discussion by nitin-pandita** [nitin-pandita/Nitin-Fashionwear#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nitin-pandita** [comment]
If you are thinking of any change you want to be implemented in the website feel free to address it here

**Comment by lionelritchie29** [comment]
Hi @nitin-pandita ,

I recently just made changes for Issue #20 , and I realized when I want to start the project locally, I didn't need to run `npm i`, since the project is in pure HTML, but the README.md were saying:
![image](https://user-images.githubusercontent.com/49032584/235855482-f7f3d01d-b1d2-4896-9974-c1d2a3d5abe7.png)

Am I missing something?

Thanks :D.

**Reply by nitin-pandita** [reply]
Yes, there is no need for the `npm i`, there is some documentation mistake, but it will be corrected. Thanks for sharing
